scio-core/src/main/scala/com/spotify/scio/coders/KryoAtomicCoder.scala

```diff
@@ -245,6 +244,7 @@ private[scio] object KryoAtomicCoder {
         k.setReferences(options.referenceTracking)
         k.setRegistrationRequired(options.registrationRequired)
 
+        new AllScalaRegistrar()(k)
```

Comment:
@ClaireMcGinty I think there's no need to include this. `AllScalaRegistrar` is already registered as part of the `ScalaKryoInstantiator`. (which is the one we use)